association_stats.go

```diff
@@ -8,13 +8,32 @@ import (
 )
 
 type associationStats struct {
+	nPackets     uint64
+	nPacketsSent uint64
```

Comment:
NIT: 
I would prefer
``` 
nPacketsReceived
nPacketsSent
```

I understand we would need to change `nSACK` to `nSACKReceived`

Reply:
I was avoiding the change but let's do it